src/HW2/divide.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+
+int main()
+{
+    int a;
+    int b;
+    scanf("%d", &a);
+    scanf("%d", &b);
+    int k = 0;
+    int e = 1;
+    if (b == 0){
```

Comment:
```suggestion
    if (b == 0) {
```
Не настроили Вы форматер. :( А надо.